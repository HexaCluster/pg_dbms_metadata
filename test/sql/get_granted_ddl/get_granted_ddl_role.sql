SELECT dbms_metadata.get_granted_ddl('ROLE_GRANT','user');