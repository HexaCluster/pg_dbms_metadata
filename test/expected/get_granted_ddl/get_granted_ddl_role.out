       get_granted_ddl       
-----------------------------
 GRANT role_test2 TO "user" +
 GRANT "Role_test" TO "user"+
 
(1 row)

